### PR TITLE
Update Sites Instructions to API Token

### PR DESCRIPTION
**Pull request by swr**
API Tokens via the profile page are the recommended auth path by the wrangler config cli and good security hygiene. It can be confusing when a new user is looking to try Sites.

Update language and instructions to direct them to create a limited token with just Workers permissions via the Workers template.

If Global API Key is still desired for the quickstart the link to the profile should be updated at least so that it points to the API section of the updated profile section.

Fixes Issues #122

### CODE REVIEW

File: src/pages/sites.js [hunk 1]
```diff
@@ -104,13 +104,14 @@ const IndexPage = () => {
...
-foo@bar.com<br/>
-Enter api key:<br/>
-123456abcdef</u></code></pre>
+                  <p>To configure Wrangler, you need a <strong>Workers API Token</strong> from your <a className="Link" href="https://dash.cloudflare.com/profile/api-tokens">Profile</a>.
+                  </p>
+                  <p>Click "Create Token" and use the "Edit Cloudflare Workers" template. For more information see <a className="Link" href="https://support.cloudflare.com/hc/en-us/articles/200167836-Managing-API-Tokens-and-Keys">Managing API Tokens and Keys
+</a></p>
+                  <p>Now run the config command to interactively enter your new API Token:</p>
+                  <pre className="CodeBlock" language="sh"><code className="CodeBlock--code"><u><b className="CodeBlock--token-directory">~/</b> <b className="CodeBlock--token-prompt">$</b> </u>wrangler config<u><br />
+Enter API Token::<br />
```
Comment (signalnerve) [inline src/pages/sites.js:113]:
```suggestion
Enter API Token:<br />
```

File: src/pages/sites.js [hunk 2]
```diff
@@ -104,13 +104,14 @@ const IndexPage = () => {
...
-                  <p>To configure Wrangler, grab your <strong>Global API key</strong> from your <a className="Link" href="https://dash.cloudflare.com/profile">Profile</a>.</p>
-                  <p>Then run the configure command and interactively enter your email address and this API key:</p>
-                  <pre className="CodeBlock" language="sh"><code className="CodeBlock--code"><u><b className="CodeBlock--token-directory">~/</b> <b className="CodeBlock--token-prompt">$</b> </u>wrangler config<u><br/>
-Enter email:<br/>
-foo@bar.com<br/>
-Enter api key:<br/>
-123456abcdef</u></code></pre>
+                  <p>To configure Wrangler, you need a <strong>Workers API Token</strong> from your <a className="Link" href="https://dash.cloudflare.com/profile/api-tokens">Profile</a>.
+                  </p>
+                  <p>Click "Create Token" and use the "Edit Cloudflare Workers" template. For more information see <a className="Link" href="https://support.cloudflare.com/hc/en-us/articles/200167836-Managing-API-Tokens-and-Keys">Managing API Tokens and Keys
```
Comment (signalnerve) [inline src/pages/sites.js:109]:
```suggestion
                  <p>Click "Create Token" and use the "Edit Cloudflare Workers" template. For more information, see <a className="Link" href="https://support.cloudflare.com/hc/en-us/articles/200167836-Managing-API-Tokens-and-Keys">"Managing API Tokens and Keys"
```

File: src/pages/sites.js [hunk 3]
```diff
@@ -104,13 +104,14 @@ const IndexPage = () => {
...
+</a></p>
```
Comment (signalnerve) [inline src/pages/sites.js:110]:
```suggestion
</a> in our documentation.</p>
```

**Comment by signalnerve** [comment]
hey @swr, was hoping that me clicking the "Update branch" button would fire off a build on our repo so the status check would work. it seems like we actually receive so few PRs from outside the cf organization that we don't actually have builds for those kinds of PRs set up - oops! regardless, this looks good to me (tested locally) and i'm merging now. thanks so much!

**Comment by swr** [comment]
Thanks. Should be all set. Great catches and tweaks. I think Sites is a great place for a lot of people to get to play with Cloudflare Workers, so making the onboarding easy was a great idea! I was literally trying to get a splash page up in as little time as possible and had been wanting to play with workers so it worked out great. Even with a fresh start it was quicker and friendlier than firebase in a lot of ways. Almost a netlify.

No worries on the build stuff, understood.

**Pull request merged**